maintainers/maintainer-list.nix

```diff
@@ -14589,4 +14589,10 @@
     github = "npatsakula";
     githubId = 23001619;
   };
+  eigengrau = {
+    email = "seb@schattenkopie.de";
+    name = "Sebastian Reuße";
+    github = "eigengrau";
+    githubId = 4939947;
+  };
```

Comment:
This should be in a separate commit.

Reply:
Hi! Could you clarify? This change is in 9712edc984d, which is already separate from the one adding the package (4d84afde092).